op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -279,7 +279,7 @@ func relayMessage(ctx context.Context, t devtest.T, source, dest *L2) error {
 				if err != nil && errors.Is(err, ctx.Err()) {
 					return ctxErrFn
 				}
-				t.Require().NoError(err)
+				requireNoErrorUnlessCancelled(t, err)
```

Comment:
Check is likely unnecessary here as well due to the check in the preceding lines.